CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Technical - New healthcheck route
```

Comment:
I think we should describe the high level feature here.
This is not a "technical" topic as it has a real value to ForestAdmin users.

```suggestion
- Session - Distinguish "CORS configuration issue" and "Server down" scenarios in case of liana login error.
```